spec/jobs/data_warehouse/stale_data_check_job_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.describe DataWarehouse::StaleDataCheckJob, type: :job do
+  let(:timestamp) { Date.new(2024, 10, 10).in_time_zone('UTC').end_of_day }
+  let(:job) { described_class.new }
+  let(:expected_bucket) { 'login-gov-analytics-export-test-1234-us-west-2' }
+  let(:test_on_tables) { ['users'] }
+  let(:s3_data_warehouse_bucket_prefix) { 'login-gov-analytics-export' }
+
+  let(:expected_json) do
+    {
+      'users' => {
+        'max_id' => 2,
+        'row_count' => 2,
+      },
+    }.to_json
```

Comment:
seems like this would be a good use for a CSV we export instead of JSON? I don't feel strongly about it but wanted to float it as an option

| table | max_id | row_count |
| --- | --- | --- |
| users | 2 | 2 |

Reply:
1. Depending on the redshfit API, if it's the async "create a query and check for results later" then you should be able to fire off a bunch of them and handle it that way
2. CSV instead of JSON is not that big of a deal, personally I think it's more legible but it's not worth the effort to bother with a transition at some undetermined point in the future
3. Seems reasonable to wait and see, yes